herd/tests/instructions/AArch64.mixed/M000.litmus

```diff
@@ -0,0 +1,14 @@
+AArch64 M000
+(* Does not work yet without option -debug mixed... *)
```

Comment:
Do you mean:
```suggestion
(* Does not work yet without option -variant mixed... *)
```

Reply:
I do mean `-debug mixed`. Undocummented `-debug mixed`  selects a less efficient but less error-prone technique to enumerate co and rf relations.